### DISCUSSION TITLE
PR2 My Open Math

### DISCUSSION THREAD

**Discussion by AbbyANoble** [TeamBasedInquiryLearning/precalculus#403] [General]
Made by @kathypinzon, but I couldn't find a discussion.

**Comment by AbbyANoble** [comment]
Looks good!

**Comment by siwelwerd** [comment]
See #397, the only comment was a typo so I fixed that and closed the issue, perhaps prematurely.

**Reply by AbbyANoble** [reply]
That was Check It and this is My Open Math discussion. :)

**Reply by AbbyANoble** [reply]
But yeah, that one was closed prior to me reviewing it!
